plugin/fulltext/mysql-test/ngram/ngram_basic.test

```diff
@@ -0,0 +1,25 @@
+# Ngram fulltext parser basic functionality.
+
+SET NAMES utf8mb4;
+SELECT @@ngram_token_size;
+
+let $engine_index= 3;
+while ($engine_index) {
+  if ($engine_index == 3) { let $engine= InnoDB; }
+  if ($engine_index == 2) { let $engine= MyISAM; }
+  if ($engine_index == 1) { let $engine= Maria; }
```

Comment:
The canonical name should be `ENGINE=Aria`.

Could we use a `.combinations` file for this, setting `default_storage_engine`? In that way, the `.result` file should contain less redundant output, and it should be trivial to ensure that each storage engine works in the same way.

Reply:
> The canonical name should be ENGINE=Aria.

Changed to `ENGINE=Aria`.

> Could we use a `.combinations` file for this, setting `default_storage_engine`?

Unlike for character sets, it's possible to use combinations for a storage engine.
However, that will add three more server restarts. As you mentioned last time, restarts take about 1-2 seconds. So I tried to reduce number of restarts. Currently `ngram_basic.result` is 421 lines long, which is still manageable. Changing to combinations will make that file about 140 lines long, at the expense of slightly increased CI run time. 

Should I change this to combinations anyway?